NuGet.config

```diff
@@ -16,7 +16,9 @@
         <add key="repositoryPath" value="..\packages" />
     </config>
   <packageSources>
+    <add key="LocallyBuiltApplicationInsightsPackages" value="..\bin\Debug\NuGet" />
```

Comment:
That's how most solutions/projects are set up, but this one isn't set up that way. The DependencyCollector has two solutions files associated with it (.../Src/Microsoft.ApplicationInsights.Web.sln and .../Test/DependencyCollector/FunctionalTests.sln). The Src solution is the one that contains the implementation and creates the DependencyCollector NuGet package. The Test solution is the one that consumes the NuGet package, and since the folder structure is so different, it can't reference the Src DependencyCollector project.